src/main/java/tr/com/srdc/cda2fhir/transform/DataTypesTransformerImpl.java

```diff
@@ -517,6 +523,10 @@ else if(ii.getRoot() != null && !ii.getRoot().isEmpty())
 		else if(ii.getExtension() != null && !ii.getExtension().isEmpty())
 			identifierDt.setValue(ii.getExtension());
 		
+		if(ii.getAssigningAuthorityName() != null) {
+			identifierDt.setAssigner(new Reference(ii.getAssigningAuthorityName()));
```

Comment:
The assigner is used to indicate what registry/state/facility/etc. assigned the identifier. As a Reference, the assigner can include just a text description in the display.

You need to create the reference then set the display object of the reference to the display value.

Reply:
ok check it out now.